airflow/utils/logging.py

```diff
@@ -211,3 +212,33 @@ def parse_gcs_url(self, gsurl):
             bucket = parsed_url.netloc
             blob = parsed_url.path.strip('/')
             return (bucket, blob)
+
+
+def get_log_filename(dag_id, task_id, execution_date, try_number):
```

Comment:
Please follow up the logging infrastructure redesign here #2422. It's certainly risky and debatable to modify the legacy code. This will definitely refactored once the logging design is finalized.